src/js/themes/base.d.ts

```diff
@@ -809,14 +833,9 @@ export interface ThemeType {
       margin?: MarginType;
     };
     options?: {
-      container?: {
-        align?: string,
-        pad?: string,
-      },
-      text?: {
-        margin?: MarginType,
-      },
-    },
+      container?: PropsOf<typeof Box>;
+      text?: PropsOf<typeof Text>;
```

Comment:
I think we only allow `margin` for `drop.options.text` other text syles won't work.

Reply:
Could use `Pick` here as well, I think.